refextract/references/kbs/report-numbers.kb

```diff
@@ -25,62 +25,93 @@ NEURO SCI	---neuro-sci
 PATT SOL	---patt-sol
 
 
-*****FermiLab*****
-<s9?9?9?9>
-<syys9?9?9s[AET ]>
-<syyyys9?9?9s[AET ]>
-
-FERMILAB CONF	---FERMILAB-Conf
+*****Fermilab extensions*****
+<s9?9?9?9(?:sAD?)?(?:sAE)?(?:sAPC)?(?:sCD)?(?:sCMS)?(?:sE)?(?:sND)?(?:sPPD)?(?:sTD?)?\b>
```

Comment:
this will match too broadly, because `s` is translated to `\s*` so it will match letter strings with no whitespace at all
with mandatory whitespace between the groups the pattern is much closer to only matching existing combinations
with that, the above still matches some combinations that don't appear in Inspire, e.g.
```
A-APC
A-CMS
A-E-T
A-ND
A-ND-T
AD-T
AE-APC
AE-ND
AE-TD
APC-ND
APC-T
CD-TD
CMS-TD
```
I think theoretically these are possible, though

Reply:
I've updated the kb to include mandatory whitespace between suffixes.

Yes, theoretically any alphabetical combo of endings is possible.